jenkins/opensearch/integ-test.jenkinsfile

```diff
@@ -3,39 +3,45 @@ lib = library(identifier: "jenkins@20211118", retriever: legacySCM(scm))
 pipeline {
     agent none
     environment {
-        AWS_ROLE_ARN = "arn:aws:iam::${AWS_ACCOUNT_PUBLIC}:role/opensearch-test"
-        AWS_ROLE_SESSION_NAME = "jenkins-test-session"
         BUILD_MANIFEST = "build-manifest.yml"
+        INTEG_TEST_JOB_NAME = 'integ-test'
```

Comment:
Instead of hard-coding you can retrieve the job_name by $JOB_NAME env variable right?

Reply:
This is a really great catch! Thanks!

The only was I've been able to test is by hardcoding the JOB_NAME  used when calling runIntegTestScript. In OpenSearch, it's not passing that value and thus able to us the correct job name but in OpenSearch Dashboards we will need to pass job name. But the assumption how JOB_NAME being used is incorrect. Will fix! Thanks.

Reply:
@kavilla - Do we want `JOB_NAME` or JOB_BASE_NAME`? I think this should be `JOB_BASE_NAME` if we only want the job name

Reply:
With the update, `JOB_NAME` is just used to upload test results to S3. Even though it's not really supported with this PR, I'd imagine if I built from my Playground pipeline (Playground/xyz) and my distribution be located in `Playground/xyz/builds`, I would expect my test results to be in `Playground/xyz/test-results`.

But this is actually super awesome as it resolves another issue I was seeing https://github.com/opensearch-project/opensearch-build/issues/1566, so thanks for that! Will create a PR for this bug so it makes it easier for me to have a playground environment.

TBH, I think most of this confusion can be mitigated with if the build manifest includes like a `JOB_NAME`. So instead of hard coding `BUILD_JOB_NAME` I can just get it it from the manifest. If we think that's a good idea I can create an issue for it. Let me know!

Thanks!